faros_event.sh

```diff
@@ -244,12 +244,48 @@ function parseFlags() {
             --run) # Externally build is referred to as run
                 run_uri="$2"
                 shift 2 ;;
+            --run_id)
```

Comment:
It seems that `--run`, `--deploy`, `--commit`, `--artifact` arguments are mutually exclusive with their fields counterparts. dont we need to check an error on it?

Reply:
We could throw an error, right now the uri just takes precedence and the other fields will be ignored.

Reply:
I am going to let the Event API do the validation it wants to do by passing through whatever we are provided. i.e. if we are given a URI and ID fields i am going to pass both on and let the API complain that only 1 should be provided. This is because I only want the validation to live in one place: the API.